packages/ord-connect/src/providers/OrdContext.tsx

```diff
@@ -123,80 +99,80 @@ export type OrdConnectProviderProps = {
  * }
  *
  * @param props - Props object.
- * @param props.initialNetwork - Initial network state.
+ * @param props.initialNetwork - Initial network state if network is not set.
  * @returns Provider component for OrdConnect.
  */
 export function OrdConnectProvider({
   children,
   initialNetwork,
 }: PropsWithChildren<OrdConnectProviderProps>) {
-  const [address, setAddress] = useState<BiAddressString>(
-    () => getItemFromLocalStorage(ADDRESS) ?? EMPTY_BIADDRESS_OBJECT,
-  );
+  if (!initialNetwork) {
+    throw new Error("Initial network cannot be empty");
+  }
 
-  const [network, setNetwork] = useState<Network>(
-    initialNetwork ?? getItemFromLocalStorage(NETWORK) ?? Network.TESTNET,
```

Comment:
@thedoublejay I changed this part here - the old code will always force `initialNetwork` to be used. If the user manually edits local storage, they can bypass `initialNetwork`. Is it fine to change this logic?

Reply:
yep